contrib/custom-node-e2e/README.md

```diff
@@ -0,0 +1,23 @@
+# Node e2e test with custom CRI-O builds
+
+The behaviour of [Kubernetes node e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md) that use [CRI-O as the CRI](https://github.com/kubernetes/test-infra/tree/master/jobs/e2e_node/crio) which run on k8s CI could be replicated locally through GCP VM(s).
+
+This could be helpful to debug node e2e test failures that would come up during developing cri-o itself. The script builds cri-o on the local system, creates an installation bundle tarball and uploads it to a GCS bucket of user's choice. While running the tests, a [GCP instance is created](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md#remotely) and the [Fedora CoreOS machine](https://github.com/kubernetes/test-infra/tree/master/jobs/e2e_node/crio/latest) consumes the output ignition config whichs installs the custom cri-o build and e2e tests are run against it.
```

Comment:
```suggestion
This could be helpful to debug node e2e test failures that would come up during developing cri-o itself. The script builds cri-o on the local system, creates an installation bundle tarball and uploads it to a GCS bucket of user's choice. While running the tests, a [GCP instance is created](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md#remotely) and the [Fedora CoreOS machine](https://github.com/kubernetes/test-infra/tree/master/jobs/e2e_node/crio/latest) consumes the output ignition config whichs installs the custom CRI-O build and e2e tests are run against it.
```

Reply:
Updated.